lib/aic.rb

```diff
@@ -81,6 +81,9 @@ class AIC < RDF::StrictVocabulary("http://definitions.artic.edu/ontology/1.0/")
   term :deptCreated,
        label: "Created by Department",
        comment: "Department of resource creator"
+  term :deptDeposited,
+       label: "Deposited by Department",
+       comment: "Department of resource depositor"
   term :digitizationSource,
```

Comment:
@awead Is this file generated? I think not b/c I couldn't find a script associated with it. It's okay to add new terms directly to this?

Reply:
Yes, that's exactly what it's for!